src/bin/vip-import-sql.js

```diff
@@ -190,44 +191,44 @@ export async function gates( app, env, fileMeta ) {
-		description: 'Import the given SQL file to your site',
+		usage: 'vip @example-app.develop import sql file.sql',
+		description:
+			'Import the local SQL database file "file.sql" to the develop environment of the "example-app" application.',
 	},
 	// `search-replace` flag
 	{
 		usage:
-			'vip import sql @mysite.develop file.sql --search-replace="https://from.example.com,https://to.example.com"',
+			'vip @example-app.develop import sql file.sql --search-replace="https://from.example.com,https://to.example.com"',
 		description:
-			'Perform a Search and Replace, then import the replaced file to your site.\n' +
-			'       * Ensure there are no spaces between your search-replace parameters',
+			'Perform a search and replace operation on the SQL database file during the import process.',
 	},
 	// `search-replace` flag
 	{
 		usage:
-			'vip import sql @mysite.develop file.sql --search-replace="https://from.example.com,https://to.example.com" --search-replace="example.com/from,example.com/to"',
+			'vip @example-app.develop import sql file.sql --search-replace="from.example.com,to.example.com" --search-replace="example.com/from,example.com/to"',
 		description:
-			'Perform multiple search and replace tasks, then import the updated file to your site.',
+			'Perform multiple search and replace operations on the SQL database file during the import process.',
 	},
 	// `in-place` flag
 	{
 		usage:
-			'vip import sql @mysite.develop file.sql --search-replace="https://from.example.com,https://to.example.com" --in-place',
+			'vip @example-app.develop import sql file.sql --search-replace="https://from.example.com,https://to.example.com" --in-place',
 		description:
-			'Search and Replace on the input `file.sql`, then import the updated file to your site',
+			'Perform a search and replace operation on "file.sql" locally, save the changes, then import the updated file.',
 	},
 	// `output` flag
 	{
 		usage:
-			'vip import sql @mysite.develop file.sql --search-replace="https://from.example.com,https://to.example.com" --output="output.sql"',
+			'vip @example-app.develop import sql file.sql --search-replace="https://from.example.com,https://to.example.com" --output="updated-file.sql"',
 		description:
-			'Output the performed Search and Replace to the specified output file, then import the replaced file to your site\n' +
-			'       * Has no effect when the `in-place` flag is used',
+			'Create a copy of the imported file with the completed search and replace operations and save it locally to a file named "updated-file.sql".',
 	},
 	// `sql status` subcommand
 	{
-		usage: 'vip import sql status @mysite.develop',
+		usage: 'vip @example-app.develop import sql status',
 		description:
-			'Check the status of the most recent import. If an import is running, this will poll until it is complete.',
+			'Check the status of the most recent SQL database file import to the develop environment of the "example-app" application.\n' +
```

Comment:
```suggestion
			'Check the status of the most recent SQL database import to the develop environment of the "example-app" application.\n' +
```

Similar to the above suggestion for this example

Reply:
Agreed!